MyClass.java

```diff
@@ -1,74 +1,71 @@
 import java.util.Scanner; //imports scanner class
 
 public class MyClass {
+	public static void main(String args[]) {
+		intro();
+		int times=getRows(); 
+		
+		int trunkNumber;
+        
+        if((1+2*times)%3 >= 2){
+            trunkNumber = (1+2*times)/3 + 1;
+        }else{
+            trunkNumber = (1+2*times)/3;
+        }
+        
+        for(int i=0; i<times; i++){
+            row(times-i,1+2*i);
+        }
+        stump((1+2*times), times);
+	}    
```

Comment:
you will need to insert the printfood(); function somewhere around here

Reply:
it's added